### ISSUE TITLE
1.17.1 Chest entity duplicates

### ISSUE THREAD

**Issue by MrBlueRay** [JamesPeters98/ChestsPlusPlus#55]
It seems that after some time the entity's that show up on chests seem to stick for example I have a gold farm set up and it shows a constant rotten flesh and gold nuggets flashing on and off, and the rotten flesh will stick even if there is nothing in the chest at the time and I believe that multiple entities start to build up in front of each chest as FPS starts to decline when in the area for the chests.

I have found that reloading Chest++ with plugman fixes the issue all be it temporarily untill the entities start to stick again, though it does seem to happen more oftern after a server restart.

I was not able to get a recording of my fps tanking in the area but I have a small clip to show the extra entities on the chests.

https://user-images.githubusercontent.com/24621768/128625180-88f9e652-b163-4640-a4dc-ea291b4f998a.mp4

**Comment by Virologicbody** [comment]
Is this is actually caused by multiplying armor stands?

I had a lot of lag which I cleared by killing all of the armor stands using _/kill @e[type=minecraft:armor_stand]_ but the lag has slowly returned.

We have a combined total of 21 chest links and auto crafters but had 1347 armor stands killed by the command!

![2021-08-13_20 34 49](https://user-images.githubusercontent.com/45987191/129411187-12a2935c-6047-4c3f-a8a5-a054b1e853cc.png)

 Now the lag has gone.

**Comment by MrBlueRay** [comment]
Yes multiplying invisible armorstands infrount of chests and autocrafters that are a result of the plugin not working as it should.

It's the same issue.

**Comment by astirling** [comment]
I'm having a similar issue on my server.  This REALLY becomes an issue when someone removes the autocrafter.  In one instance, I had 5,000 invisible armor stands lagging my server where a player had previously had an autocrafter.

**Comment by JamesPeters98** [comment]
I think this may be fixed in the next release. I will release a Beta later today. If you join the Discord you can find it there.
